scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -169,7 +169,11 @@ def uberenv_build(prefix, spec, project_file, mirror_path):
     if project_file:
         cmd += "--project-json=\"{0}\" ".format(project_file)
 
-    spack_tpl_build_log = pjoin(prefix,"output.log.spack.tpl.build.%s.txt" % spec.replace(" ", "_"))
+    # Take the first 200 characters if spec is too long
```

Comment:
Does it cause an error if it's too long? Or is this just for readability?
(A comment in the file might help)